web/cobrands/lincolnshire/assets.js

```diff
@@ -25,6 +25,13 @@ fixmystreet.assets.add(defaults, {
     asset_item: 'bollard'
 });
 
+fixmystreet.assets.add(defaults, {
+    wfs_url: "https://tilma.staging.mysociety.org/mapserver/lincs",
```

Comment:
This line will be removed once the layers are on live tilma obvs :)

Reply:
If staging has all the layers on as well, you could copy what other assets.js do and set the server based upon fixmystreet.staging, then you don't need to remember (well, remember this, you do need to remember to put the assets on live as well then).